templates/publisher/_beta-notification.html

```diff
@@ -0,0 +1,12 @@
+<section class="p-strip is-shallow u-no-padding--bottom" id="beta-notification">
+  <div class="row">
+    <div class="p-notification--note">
+      <div class="p-notification__response">
+        <button class="p-icon--close u-float--right" aria-label="Close notification" onclick="document.getElementById('beta-notification').remove()">Close</button>
```

Comment:
Not a critical thing, but `remove` may not be supported in some browsers (IE and mobile Safari) which may lead to some exceptions (sentry).

Reply:
Maybe just `style.display = 'none'` instead, do you think?

Reply:
This is now not needed